libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -711,6 +724,23 @@ bool AP_Airspeed::all_healthy(void) const
     return true;
 }
 
+#else  // build type is not appropriate; provide a dummy implementation:
+const AP_Param::GroupInfo AP_Airspeed::var_info[] = { AP_GROUPEND };
+
+bool AP_Airspeed::get_temperature(uint8_t i, float &temperature) { return false; }
+void AP_Airspeed::calibrate(bool in_startup) {}
+bool AP_Airspeed::use(uint8_t i) const { return false; }
+
+#if HAL_MSP_AIRSPEED_ENABLED
+void AP_Airspeed::handle_msp(const MSP::msp_airspeed_data_message_t &pkt) {}
+#endif
+
+bool AP_Airspeed::all_healthy(void) const { return false; }
+void AP_Airspeed::init(void) {};
+AP_Airspeed::AP_Airspeed() {}
+
+#endif // #if !(APM_BUILD_COPTER_OR_HELI && BOARD_FLASH_SIZE <= 1024) && !APM_BUILD_TYPE(APM_BUILD_AntennaTracker) && !APM_BUILD_TYPE(APM_BUILD_Blimp)
```

Comment:
```suggestion
#endif // #if AP_AIRSPEED_DUMMY_METHODS_ENABLED
```

Reply:
Done. Thank you for this! I was having the darnedest time on this one.